services/galley/src/Galley/API/Error.hs

```diff
@@ -210,7 +210,7 @@ legalHoldCouldNotBlockConnections :: Error
 legalHoldCouldNotBlockConnections = mkError status500 "legalhold-internal" "legal hold service: could not block connections when resolving policy conflicts."
 
 missingLegalholdConsent :: Error
-missingLegalholdConsent = mkError status412 "missing-legalhold-consent" "Failed to connect to a user or to invite a user to a group because somebody is under legalhold and somebody else has not granted consent."
+missingLegalholdConsent = mkError status403 "missing-legalhold-consent" "Failed to connect to a user or to invite a user to a group because somebody is under legalhold and somebody else has not granted consent."
```

Comment:
To avoid this duplication, you could turn this into an `ErrorDescription` and move it to the `ErrorDescription` module in wire-api, where it can then be used from both brig and galley. Also, it makes it easier to add it to swagger, if desired, or even later turn it into a statically checked response with `MultiVerb`.

I've done this for a bunch of errors as part of https://github.com/wireapp/wire-server/pull/1657. This is a typical example: https://github.com/wireapp/wire-server/pull/1657/commits/8e73769d7d4a657fef5fbe9210f885f9ccbcaab6. It can be a bit of work though, depending on how often the error is thrown, so feel free to ignore this.

Reply:
thanks, i'll look into it.

Reply:
(why is nothing ever easy?  :-))

Reply:
I've solved it like this: 83abb9cae85112193dac8b9e976a7bc8a70612ff

I've made you the commit author @pcapriotti, let me know if that's not ok and I'll change it back.

Reply:
Sure, that's fine.

Reply:
FYI: #1693 contains the change I was suggesting, since it was useful for another refactoring.